packages/@webex/plugin-meetings/test/unit/spec/meeting/brbState.ts

```diff
@@ -0,0 +1,112 @@
+import sinon from 'sinon';
+import {assert} from '@webex/test-helper-chai';
+
+import testUtils from '../../../utils/testUtils';
+import {BrbState, createBrbState} from '@webex/plugin-meetings/src/meeting/brbState';
+
+describe('plugin-meetings', () => {
+  let meeting: any;
+  let brbState: BrbState;
+
+  beforeEach(async () => {
+    meeting = {
+      isMultistream: true,
+      locusUrl: 'locus url',
+      deviceUrl: 'device url',
+      selfId: 'self id',
+      mediaProperties: {
+        webrtcMediaConnection: true,
+      },
+      sendSlotManager: {
+        setSourceStateOverride: sinon.stub(),
+      },
+      meetingRequest: {
+        setBrb: sinon.stub().resolves(),
+      },
+    };
+
+    brbState = new BrbState(meeting, false);
+    await testUtils.flushPromises();
+  });
+
+  describe('brbState library', () => {
+    it('takes into account current status when instantiated', async () => {
+      // create a new BrbState instance
+      brbState = createBrbState(meeting, true);
+      await testUtils.flushPromises();
+
+      assert.isTrue(brbState.isEnabled());
+
+      // now check the opposite case
+      brbState = createBrbState(meeting, false);
+      await testUtils.flushPromises();
+
+      assert.isFalse(brbState.isEnabled());
+    });
+
+    it('can be enabled', async () => {
+      brbState.enable(true, meeting.sendSlotManager);
+      brbState.handleServerBrbUpdate(true);
+      await testUtils.flushPromises();
+
+      assert.isTrue(brbState.isEnabled());
+    });
+
+    it('can be disabled', async () => {
+      brbState.enable(false, meeting.sendSlotManager);
+      brbState.handleServerBrbUpdate(false);
+      await testUtils.flushPromises();
+
+      assert.isFalse(brbState.isEnabled());
+    });
+
+    it('won\t be able to send local brb state to server if not a multistream meeting', async () => {
```

Comment:
Nit: two suggestions

```suggestion
    it('won\'t be able to send local brb state to server if it is not a multistream meeting', async () => {
``` 
```suggestion
    it('will not be able to send local brb state to server if it is not a multistream meeting', async () => {
```